assets/lang/sklmessages_pl_PL.properties

```diff
@@ -1,26 +1,26 @@
 # Contributors History:
 # skmedix: 24.08.2017 - ????
 
-lang.name=Polski
+lang.name=polski
 lang.code=pl_PL
 
 tab.news=Aktualno\u015bci
-tab.log=Logi Launchera
-tab.output=Logi Gry (%s)
-tab.crash=Raport Crasha (%s)
+tab.log=Dziennik launchera
+tab.output=Dziennik Gry (%s)
```

Comment:
The second word should also be lowercase

Reply:
Oh I forgot